src/cata_tiles.h

```diff
@@ -725,4 +736,13 @@ class cata_tiles
         std::string memory_map_mode = "color_pixel_sepia";
 };
 
+struct cata_tiles_test_helper {
```

Comment:
The definition for this should just be able to go in the tests, if you want.

Reply:
And it will be found to make it a friend class of `cata_tiles`? Would have thought not.

Reply:
Yep. I would have thought you need to forward declare it, but not even that. I can't find any source to back it up, but it works for me locally.
All I can find is 
> The name of the class that is used in this friend declaration does not need to be previously declared.

From https://en.cppreference.com/w/cpp/language/friend

Reply:
Ah, cool! Thanks! Will do...

Reply:
Done.